@xen-orchestra/lite/src/App/TreeView.tsx

```diff
@@ -52,6 +52,7 @@ const TreeView = withState<State, Props, Effects, Computed, ParentState, ParentE
           const hosts = state.hostsByPool
             ?.get(pool.$id)
             ?.valueSeq()
+            .sortBy((host: host) => host.name_label)
```

Comment:
```suggestion
            .sortBy(host => host.name_label)
```

You don't need to specify a type because he is already defined [here](https://github.com/vatesfr/xen-orchestra/blob/xo-lite/%40xen-orchestra/lite/src/App/TreeView.tsx#L24).
You can also remove the definition type for the `.map` below.